library/src/interactor_impl.cxx

```diff
@@ -938,6 +962,23 @@ interactor& interactor_impl::initBindings()
     valStream << val;
     return std::pair(doc, valStream.str());
   };
+  
+  // "doc", "value/Unset"
+  auto docDblOpt = [](const std::string& doc, const std::optional<double>& val)
+  {
+    std::stringstream valStream;
+    valStream.precision(2);
+    valStream << std::fixed;
+    if (val.has_value())
+    {
+      valStream << val.value();
```

Comment:
Ideally you may also want to add/modify a test that show the cheatsheet with the opacity modified in order to cover  this line.

Adding a "TestInteractionCheatsheetOpacity" comes to mind. the log only need to press "H" to show the cheatsheet, the rest can be command line args.